### ISSUE TITLE
Request: Disable avatars

### ISSUE THREAD

**Issue by telebiker** [moezbhatti/qksms#1750]
Hello!

Unfortunately #1030 and #1563 was closed with no resolution. I know that it is Open Source and nobody has to fullfil my requests, but I'm using this messenger because it had an option not to show avatars, but for some reason this option was removed. Yes, I can read. I don't need these images, they waste screen space only (I don't have photos attached to contacts at all). Also, showing avatars is the feature, not hiding them, and now this feature is turned on forcibly. I would be grateful if the option to hide avatars is enabled back. I don't know any other SMS clients who can do that.

Thank you very much.

@Bill-Stewart @MyNameIsCosmo

**Comment by moezbhatti** [comment]
The resolution is wontfix – there just aren't enough users who care about having the ability to hide avatars. And when I didn't implement this setting in the 3.0 rewrite, I had data from the old version of the app to prove that it wasn't being used. I don't have this data anymore so I can't give you an exact number, but if I recall correctly, less than 1% of users disabled contact avatars in QKSMS 2.0

As of QKSMS 3.0, I only want to add settings for features that either
1. Are required by certain users to make the app functional
2. I expect will be useful for at least 10% of users

Otherwise, the settings screen gets cluttered and hard to navigate. QKSMS isn't aiming to have every setting and customization under the sun – there are other apps like YAATA or Textra for that. QKSMS is designed to be intuitive, beautiful, and work well for most people right out of the box

**Comment by telebiker** [comment]
Thanks for taking care of reading users' settings! Really important stuff.

**Comment by moezbhatti** [comment]
The Google Play release of QKSMS collects anonymous usage data so that I can understand which parts of the app are actually used, and what kind of work I should be prioritizing.

**Comment by telebiker** [comment]
Removing features which were available without any reason does not make any sense even if they were used by small amount of people.

**Comment by moezbhatti** [comment]
QKSMS 3.0 was a rewrite from the ground up. I built everything from scratch. It was basically a whole new app

I understand that from your perspective it was removed, but I was building everything from scratch and I had to decide whether or not I should build it again. One of the most frequent complaints I received about QKSMS 2.0 was that important settings were hard to find, and the amount of settings was overwhelming for users who were not power-users. I used this feedback, along with the data about which settings were being used in the first place, and built an app that better suited the needs of most of its users

If you're unhappy with my decision, you're more than welcome to stick to QKSMS 2.0. The APK is available here: https://github.com/moezbhatti/qksms/releases/tag/v2.7.3
